### DISCUSSION TITLE
BLTouch probing twice

### DISCUSSION THREAD

**Discussion by scaramonga** [Jyers/Marlin#873] [Q&A] [answered]
What is the setting to have the BLTouch probe the bed only once?  At the minute, its 5x5, which is fine, but it does each spot twice, which is unnecessary.  I'd like to have it do each just once.

Thx folks :)

**Comment by smooreace** [comment]
Its only sampling once. The first run is fast is only to find about where
the bed it. the second one is slow and actually measuring the bed

**Comment by gothcha** [comment, marked as answer]
@scaramonga If you wish to disable the double probing then comment out this statement in configuration.h

#define MULTIPLE_PROBING 2

I also changed the speed

#define Z_PROBE_FEEDRATE_FAST (6*60)

I run my setup with this fast single probing very happily. Worth a try.

**Reply by scaramonga** [reply]
Nice!  I'll give that a try :)  Thanking you kindly.
